src/components/Treenav/TreenavCategory.tsx

```diff
@@ -30,6 +44,22 @@ export const TreenavCategory = ({ guide, index, open }) => {
     }
   };
 
+  const onSectionClick = async (
+    event: React.MouseEvent,
+    section: SectionModel
+  ) => {
+    event.preventDefault(); // I've seen this used for a lot of onClick methods, does this do anything here/wanna explain what it is? LOL
+    router.push({
+      pathname:
+        '/cookbooks/[cookbookId]/recipes/[recipeId]/sections/[sectionId]',
```

Comment:
Just in general, working with slugs might be preferable so users know what content to expect when you send them the link. Possibly better for future work

Reply:
yeah, i agree, should be used for future work tho

we had some annoying stuff using slugs earlier but i dont think thats a reason to leave it :)

Reply:
Uh we can talk about it but Im actually super against slugs